### PR TITLE
cmd/swarm,pss,swap: fixed linter errors S1038/S1039

### LINKED ISSUE #2135: Linter fails on fmt.Print --> S1038/S1039 clauses
Linter fails to due new static checks

Linter errors 
- [S1038](https://staticcheck.io/docs/checks#S1038)
- [S1038](https://staticcheck.io/docs/checks#S1038)

This should be updated ASAP

### PR DESCRIPTION
**Pull request by santicomp2014**
This resolves #2135 

Fixed linter issues

### CODE REVIEW

File: cmd/swarm/main.go [hunk 1]
```diff
@@ -283,10 +283,10 @@ func keys(ctx *cli.Context) error {
 	bzzkey := crypto.Keccak256Hash(pubkey).Hex()
 
 	if !ctx.Bool(SwarmCompressedFlag.Name) {
-		fmt.Println(fmt.Sprintf("bzzkey=%s", bzzkey[2:]))
-		fmt.Println(fmt.Sprintf("publicKey=%s", pubkeyhex))
+		fmt.Print(fmt.Sprintf("bzzkey=%s", bzzkey[2:]), "\n")
+		fmt.Print(fmt.Sprintf("publicKey=%s", pubkeyhex), "\n")
 	}
-	fmt.Println(fmt.Sprintf("publicKeyCompressed=%s", pubCompressed))
+	fmt.Print(fmt.Sprintf("publicKeyCompressed=%s", pubCompressed), "\n")
```
Comment (janos) [inline cmd/swarm/main.go]: Could these lines be in a simpler form `fmt.Printf("...%s\n", ARGS...)`?

**Review by janos — APPROVED** [review]
It appears that linter got improved. Thanks for fixing issues. I have only one minor comment that is not crucial. LGTM.

File: cmd/swarm/main.go (hunk 1 above)
Comment (santicomp2014) [reply cmd/swarm/main.go]: Fixed

**Review by ralph-pichler — APPROVED** [review]
LGTM

**Review by acud — APPROVED** [review]

**Pull request merged**